index.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+// Key in annotations object that maps to an executor NPM module
+const ANNOTATION_EXECUTOR_TYPE = 'beta.screwdriver.cd/executor';
```

Comment:
`beta.screwdriver.cd` only makes sense for our cluster. Should it be more general?